api/proto/v1/common.proto

```diff
@@ -19,6 +19,7 @@ enum OperationType {
     CREATE = 1;
     UPDATE = 2;
     DELETE = 3;
+    CLEAN = 4;
   }
```

Comment:
@phudtran Does this operation require Scenario Manager sending down the operation type to Compute?

Reply:
Yes it does.